modules/op-release-notes-1-11.adoc

```diff
@@ -0,0 +1,200 @@
+
+* With this update, you can extend the scope of the GitHub token at the following two levels:
+** Repository-level: Use this level to extend the scope to the repositories that exist in the same namespace in which the original repository exists.
+** Global-level: Use this level to extend the scope to the repositories that exist in a different namespace.
+
+* With this update, {pac} triggers a CI pipeline for a pull request created by a user who is not an owner, collaborator, or public member or is not listed in the `owner` file but has permissions to push changes to the repository.
+
+* With this update, the custom console setting allows you to use custom parameters from a `Repository` CR. 
+
+* With this update, {pac} changes all `PipelineRun` labels to `PipelineRun` annotations. You can use a `PipelineRun` annotation to mark a Tekton resource, instead of using a `PipelineRun` label.
+
+* With this update, {pac} provides an option to install a webhook forwarder named `gosmee`. This forwarder enables connectivity between the {pac} controller and GitHub without requiring an internet connection. You need to explicitly specify the `--force-gosmee` flag when bootstrapping on the {product-title} cluster.
+
+
+[id="breaking-changes-1-11_{context}"]	
+== Breaking changes	
+
+* This update replaces the `resource-verification-mode` feature flag with a new flag named `trusted-resources-verification-no-match-policy` in the pipeline specification.
+
+
+[id="deprecated-features-1-11_{context}"]	
+== Deprecated and removed features	
+
+* This update removes support for the `PipelineResource` commands and references from Tekton CLI:
+** Removal of `PipelineResources` from cluster tasks
+** Removal of `PipelineResources` from tasks
+** Removal of `PipelineResources` from pipelines
+** Removal of resource commands
+** Removal of input and output resources from the `clustertask describe` command 
+
+* This update removes support for the `full` embedded status from Tekton CLI.
+
+* In {pipelines-title} 1.11, support for the `PipelineResource` CR has been removed, use the `Task` CR instead.
+
+* In {pipelines-title} 1.11, support for the `v1alpha1.Run` objects has been removed. You must upgrade the objects from `v1alpha1.Run` to `v1beta1.CustomRun` before upgrading to this release. 
+
+* In {pipelines-title} 1.11, the `custom-task-version` feature flag has been removed.
+
+* In {pipelines-title} 1.11, the `pipelinerun.status.taskRuns` and `pipelinerun.status.runs` fields have been removed along with the `embedded-status` feature flag. As an alternative, you can use the `pipelinerun.status.childReferences` field instead.
+
+
+[id="known-issues-1-11_{context}"]	
+== Known issues	
+
+* Setting the `prune-per-resource` boolean field does not delete `PipelineRun` or `TaskRun` resources, if they are not defined in a pipeline or task. 
```

Comment:
I'm confused about what this means...pipeline/task don't define pipelinerun/taskrun :confused:

Reply:
Modified for clarity: Setting the `prune-per-resource` boolean field does not delete `PipelineRun` or `TaskRun` resources, if they did not belong to any pipeline or task.